### PR TITLE
setup docs

### PR DESCRIPTION
**Pull request by guilhermebodin**

### CODE REVIEW

File: docs/make.jl [hunk 1]
```diff
@@ -0,0 +1,20 @@
+using Documenter, Dualization
+
+makedocs(
+    modules = [Dualization],
+    doctest  = false,
+    clean    = true,
+    format   = Documenter.HTML(),
```
Comment (blegat) [inline docs/make.jl]: I would recommend using these options: https://github.com/JuliaOpt/MathOptInterface.jl/blob/master/docs/make.jl#L5-L8

File: docs/Project.toml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[deps]
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+
+[compat]
+Documenter = "~0.21"
```
Comment (blegat) [inline docs/Project.toml]: Why not 0.22 ?

Comment (guilhermebodin) [reply docs/Project.toml]: No reason, I am going to change it

File: docs/make.jl (hunk 1 above)
Comment (blegat) [reply docs/make.jl]: You have twice the field format now

Comment (guilhermebodin) [reply docs/make.jl]: hehehe ça arrive

**Pull request merged**